specs/app.spec.js

```diff
@@ -0,0 +1,149 @@
+const fs = require('fs'),
```

Comment:
The program should be able to use any file you specify that is relative to the root directory... even something crazy like ../../../windows/system32/hello.csv